### PR TITLE
syncthing: add runit for user

### PR DESCRIPTION
**Pull request by Bridouz**
As syncthing should not be launched by `root` I found a way to start `syncthing` with `user rights`.

`run` must be filled with your username.

### CODE REVIEW

File: srcpkgs/syncthing/files/syncthing/run [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+[ -r conf ] && . ./conf
+
+export USERNAME=
+export HOME="/home/$USERNAME"
```
Comment (Duncaen) [inline srcpkgs/syncthing/files/syncthing/run]: Those variables should be defined before sourcing `conf` to allow using the config file to define them without editing the `run` file which gets overwritten on updates.

**Comment by Duncaen** [comment]
I still think its not the greatest way to add user services but this is requested a lot and there is no way to run multiple syncthing instances without also editing syncthings config anyways.

https://github.com/voidlinux/void-packages/issues/6314#issuecomment-408665715

**Comment by Bridouz** [comment]
Hi,

Sorry but I can't find how to do it. I tried to add a `conf` file with :

```
USERNAME="my_user"
HOME="/home/$USERNAME"
```

`run` file is : 

```
#!/bin/bash
[ -r conf ] && . ./conf

exec 2>&1
exec chpst -u "$USERNAME" syncthing -logflags 0
```

When I launch the script it says `chpst: fatal: unknown user/group:`

**Comment by Duncaen** [comment]
The conf file has to be in the service directory e.g. `/etc/sv/syncthing`, the service executes the `run` script from inside of this directory.

Try this: 

```
cd /etc/sv/syncthing
sh -x run
```

You still need to export `HOME` as far as I know.

**Comment by the-maldridge** [comment]
Closed at the request of the author.

**Pull request closed without merging**